config/systemConfig/javascript.yaml

```diff
@@ -9,4 +9,16 @@ systemConfig:
+
+  - key: apiGraphqlReadSink
```

Comment:
Instead of adding so much in config - why not a rule in internal APIs ?
rule = "{apiGraphqlLibraries}.*{apiGraphqlReadSink}.*"

Reply:
Rule will have only one pattern but we need multiple patterns here.